scripts/tests/unit/test_schema_loader.py

```diff
@@ -646,6 +646,167 @@ def test_merge_non_array_attributes(self):
 
+    def test_merge_multi_fields(self):
+        schema1 = {
+            'base': {
+                'field_details': {
+                    'multi_fields': [
+                        {
+                            'type': 'text',
+                            'name': 'text'
+                        },
+                        {
+                            'type': 'keyword',
+                            'name': 'caseless',
+                            'normalizer': 'lowercase'
+                        }
+                    ]
+                },
+                'fields': {
+                    'message': {
+                        'field_details': {
+                            'multi_fields': [
+                                {
+                                    'type': 'text',
+                                    'name': 'text'
+                                }
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+
+        schema2 = {
+            'base': {
+                'field_details': {
+                    'multi_fields': [
+                        {
+                            'type': 'text',
+                            'name': 'text'
+                        },
+                        {
+                            'type': 'text',
+                            'name': 'almost_text',
+                        }
+                    ]
+                },
+                'fields': {
+                    'message': {
+                        'field_details': {
+                            'multi_fields': [
+                                {
+                                    'type': 'keyword',
+                                    'name': 'a_field'
+                                }
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+        merged_fields = loader.merge_fields(schema1, schema2)
+        expected_multi_fields = [
+            {
+                'type': 'text',
+                'name': 'almost_text'
+            },
+            {
+                'type': 'keyword',
+                'name': 'caseless',
+                'normalizer': 'lowercase'
+            },
+            {
+                'type': 'text',
+                'name': 'text'
+            }
+        ]
+
+        expected_message_multi_fields = [
+            {
+                'type': 'keyword',
+                'name': 'a_field'
+            },
+            {
+                'type': 'text',
+                'name': 'text'
+            }
+        ]
+        self.assertEqual(merged_fields['base']['field_details']['multi_fields'], expected_multi_fields)
+        self.assertEqual(merged_fields['base']['fields']['message']['field_details']
+                         ['multi_fields'], expected_message_multi_fields)
+
+    def test_overwrite_multi_fields(self):
```

Comment:
If you agree with the suggested adjustments for the test case above, I think this makes this second test case redundant. WDYT?

Reply:
👍